apps/package.json

```diff
@@ -4,7 +4,7 @@
   "repository": "code-dot-org/code-dot-org",
   "license": "SEE LICENSE IN LICENSE",
   "engines": {
-    "node": ">=6.7",
+    "node": ">=8.15",
     "npm": "^3.10.8"
```

Comment:
the npm version should be updated as well; node 8 uses I believe npm 6

Reply:
Discussed offline. I'm going to keep the version of npm as is for now. The upgrade to node 10 will require an upgrade to npm 6.